Clarifying CIDR usage (#10064)
DataDog/documentation: content/en/synthetics/guide/identify_synthetics_bots.md

https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
```

> **Note:** IPs are listed using the CIDR notation. You might or might not need to convert them to IPv4 Address Range before using them.

Author: ```suggestion
**Note:** IPs are listed using the CIDR notation. You might or might not need to convert them to IPv4 Address Ranges before using them.
```

> **Note:** IPs are listed using the CIDR notation. You might or might not need to convert them to IPv4 address ranges before using them.

Reviewer: ```suggestion
**Note:** IPs listed use the CIDR notation and may require conversion to IPv4 address ranges before use.
```
